components/motor/gpio/loop.go

```diff
@@ -9,7 +9,7 @@ import (
 // SetState sets the state of the motor for the built-in control loop.
 func (m *EncodedMotor) SetState(ctx context.Context, state []*control.Signal) error {
 	power := state[0].GetSignalValueAt(0)
-	return m.SetPower(ctx, power, nil)
+	return m.SetPower(ctx, power/255, nil)
```

Comment:
comment on why this is needed.

Reply:
adding a comment about how it's to scale the power input over the 0 to +/- 255 range from the control config, but is it valid to assume that this 255 is correct? in the docs we just say to use +/- 255 as the max/min, but a user definitely could change that

Reply:
although this whole project has kind of been under the assumption they are using the bucket rover so maybe it's fine